### PR TITLE
Auto-correct all casks with RuboCop-Cask 0.2.0 and RuboCop master

### PR DESCRIPTION
**Pull request by jawshooah**
This is a big one, over 1200 file changes. But I trust in my specs, and I think you'll get the gist of what's been done after looking at `.rubocop.yml` and a few of the casks.

I've updated our `rubocop` dependency to point at the current `master` (it's fixed in `Gemfile.lock`, so no unexpected updates), but there should be a release with all the necessary changes within a week or so.

### CODE REVIEW

File: Casks/adobe-digital-editions.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask 'adobe-digital-editions' do
   version '4.0'
-  sha256 :no_check    # required as upstream package is updated in-place
```
Comment (vitorgalvao) [inline Casks/adobe-digital-editions.rb]: It even fixes this spacing? Huge thumbs up.

**Comment by adityadalal924** [comment]
Travis is going to time out, but everything seems in order :+1:

**Comment by vitorgalvao** [comment]
This is a happy PR. So good to see so many fixes.

**Comment by adityadalal924** [comment]
Also might want to open up an issue and tag it as `future` to revert to a tagged release once the next release of rubocop comes out.

**Comment by jawshooah** [comment]
@vitorgalvao I figured you'd be happy to see [this](https://github.com/jawshooah/homebrew-cask/blob/5dc664d68685d9ae977b886323741ee12ea93e0a/.rubocop.yml#L61-L62) :wink:

**Comment by jawshooah** [comment]
And just so we're all clear, [rubocop-cask](https://github.com/caskroom/rubocop-cask) is a @caskroom/maintainers family project. Everyone's invited to pitch in.

I find it handy to always have tabs open with the rubydocs for [rubocop](http://www.rubydoc.info/gems/rubocop/Astrolabe/Node), [astrolabe](http://www.rubydoc.info/gems/astrolabe/Astrolabe/Node), [parser](http://www.rubydoc.info/gems/parser/Parser/Source/Map), and [ast](http://www.rubydoc.info/gems/ast/AST/Node).

**Comment by jawshooah** [comment]
Travis failure was from a botched merge conflict resolution during the original rebase. Just did a sanity check on the file (`hocus-focus.rb`) to make sure it wasn't a weird edge case, and everything worked fine.

**Comment by vitorgalvao** [comment]
> @vitorgalvao I figured you'd be happy to see [this](https://github.com/jawshooah/homebrew-cask/blob/5dc664d68685d9ae977b886323741ee12ea93e0a/.rubocop.yml#L61-L62) :wink: 

I am. I do like when things in programming are enclosed with matching characters (as opposed to using the same character to open and close).

**Comment by jawshooah** [comment]
As @adityadalal924 predicted, Travis timed out during the audit phase. Is anyone opposed to merging this now?

**Comment by adityadalal924** [comment]
Cask to update after merging (download URL changed or shasum silently updated):

```
blackvue-viewer
blast2go
cmpl
daemon-tools-lite
dbgl
minimetro
navicat-for-mariadb
navicat-for-mysql
navicat-for-oracle
navicat-for-postgresql
navicat-for-sql-server
navicat-for-sqlite
navicat-premium-essentials
padre
qq
```

@jawshooah I would go ahead and merge, we can always fix things like the above after the fact (as this PR didn't break them in the first place)

**Comment by jawshooah** [comment]
Agreed.

I need to rework `audit_modified_casks` though. It should only download if the values of `version`, `sha256`, or `url` have actually changed, not just if they show up in the file diff.

**Comment by adityadalal924** [comment]
PRs incoming for the funky Casks, in any case.

**Comment by jawshooah** [comment]
:+1: Good call, thanks!

Another benefit of this merge: we now have a baseline to start applying RuboCop to the core library!

**Comment by adityadalal924** [comment]
Casks fixed in PRs https://github.com/caskroom/homebrew-cask/pull/16320 to https://github.com/caskroom/homebrew-cask/pull/16333

**Pull request merged**